framework/wazuh/agent.py

```diff
@@ -1513,3 +1513,19 @@ def get_full_overview() -> WazuhResult:
               'agent_version': stats_version, 'last_registered_agent': last_registered_agent}
 
     return WazuhResult({'data': result})
+
+
+@expose_resources(actions=["agent:uninstall"], resources=["*:*:*"], post_proc_func=None)
+def agent_check_uninstall_permission() -> WazuhResult:
```

Comment:
```suggestion
def check_uninstall_permission() -> WazuhResult:
```

Reply:
Fixed in efb4a691b97c462325ee2b7e9c4faa616a435692.